Modules/SubModule_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -39,47 +43,65 @@ function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux)
         PathToSearch = xASL_adm_UnixPath(ROOT);
         system(['for i in `find ' PathToSearch '/* | grep -E \.nii$`; do gzip -1 -f -q -v "$i"; done'], '-echo');
         return;
+    else
+        %% 2) Otherwise use the multithreaded SuperGzip for Windows
+        PathList = xASL_adm_GetFileList(ROOT, '^.*\.(nii)$', 'FPListRec', [0 Inf], false);
+        fprintf('\n%s\n',['G-zZzZipping ' num2str(length(PathList)) ' files']);
+        n_THREADS = min([str2num(getenv('NUMBER_OF_PROCESSORS')) length(PathList)]);
```

Comment:
We use camelcase `nThreads` as style throughout ExploreASL (as Matlab does)

Reply:
also this should be replaced by my suggestion in the issue, NUMBER_OF_PROCESSORS doesn't work in all cases

Reply:
Should be fixed 👍